docs/hardening.md

```diff
@@ -85,6 +85,11 @@ kubelet_streaming_connection_idle_timeout: "5m"
 kubelet_make_iptables_util_chains: true
 kubelet_feature_gates: ["RotateKubeletServerCertificate=true","SeccompDefault=true"]
 kubelet_seccomp_default: true
+kubelet_systemd_hardening: true
+# in case you have multiple interfaces in your
+# control plane nodes and you want to specify the right
+# IP addresses
+kubelet_secure_address: "192.168.10.110 192.168.10.111 192.168.10.112"
```

Comment:
The variable name is singular and the example is in the plural, this is confusing, the example should make it clear how to set these either on a per node basis or rename the variable to the plural and explain that the variable is a list of addresses from the `secure` interface of each node.

Reply:
Yes, probably the variable should be renamed in `kubelet_secure_addresses`.